SecurityCenter/src/V1/resources/security_center_rest_client_config.php

```diff
@@ -226,52 +226,5 @@
                 ],
             ],
         ],
-        'google.longrunning.Operations' => [
```

Comment:
@dwsupplee I still see Operations used in this client. Think we need to add these bindings back?

Reply:
Yes, it looks like we do. There is an issue upstream that has been stripping out these operation bindings. For now, would you be able to help patch these back in with synthtool or something similar?